source/simplex/barycentric_polynomials.cc

```diff
@@ -0,0 +1,334 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2020 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+
+#include <deal.II/simplex/barycentric_polynomials.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+namespace Simplex
+{
+  namespace internal
+  {
+    /**
+     * Get the highest degree of the barycentric polynomial (in Cartesian
+     * coordinates).
+     */
+    template <int dim>
+    unsigned int
+    get_degree(const std::vector<BarycentricPolynomial<dim>> &polys)
+    {
+      // Since the first variable in a simplex polynomial is, e.g., in 2D, 1 - x
```

Comment:
*Since the first variable in a simplex polynomial is* is what?

Reply:
I think I made this clearer - do you see the example now?